spec/models/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe User do
+
+ it 'authenticates when given a valid email address and password' do
+   user = create :user
+   authenticated_user = User.authenticate(user.email, user.password)
+   expect(authenticated_user).to eq user
+ end
+
+ it 'does not authenticate when given an incorrect password' do
+  user = create :user
+   expect(User.authenticate(user.email, 'wrong_stupid_password')).to be_nil
+  end
```

Comment:
Lint/BlockAlignment: `end` at 12, 2 is not aligned with `it 'does not authenticate when given an incorrect password' do` at 9, 1